lib/logflare/backends/adaptor/webhook_adaptor.ex

```diff
@@ -1,21 +1,21 @@
 defmodule Logflare.Backends.Adaptor.WebhookAdaptor do
   @moduledoc false
   use GenServer
-  alias Logflare.Backends.{SourceBackend, Adaptor, Adaptor.WebhookAdaptor, SourceDispatcher}
-  alias Logflare.{Backends, Buffers.MemoryBuffer}
-  use Adaptor
```

Comment:
why remove the macro?
the macro is supposed to stub certain functions (for example, throw on endpoint query, since this is ingest only.)

Reply:
we're using the adaptor, just removed the alias but I will add it back so it can be properly called by typespecs